.azure-pipelines/generate-v1.0-sdk.yml

```diff
@@ -0,0 +1,172 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+name: $(BuildDefinitionName)_$(SourceBranchName)_$(Date:yyyyMMdd)$(Rev:.r)
+
+trigger: none # disable triggers based on commits.
```

Comment:
nitpick: Add note that this pipeline is intended to be triggered only after a metadata has been successfully cleaned and captured. It is uses another pipeline as a trigger.